doc/source/tutorial/tutorial.ipyml

```diff
@@ -0,0 +1,2918 @@
+cells:
+
+- code: |
+    # @tutorial writers: You can remove cells from the HTML/LaTeX output by adding this to the cell metadata:
```

Comment:
I would be nice if we could have this cell hidden in binder (but ideally silently executed), like in the doc.